src/storage/inventory.cpp

```diff
@@ -49,8 +49,8 @@ static std::string pciVenorLookup(const std::string& aVendorId)
 }
 
 StorageDrive::StorageDrive(sdbusplus::bus::bus& bus, const std::string& aName,
-                           const std::string& aPath, const std::string& aType,
-                           const std::string& aVendor,
+                           const std::string& aPath, const std::string& aProto,
```

Comment:
I was sure that the Hungarian notation was prohibited in OpenBMC.
What is the 'a' prefix?

Reply:
That is just to have different names for arguments and class members - I don't like when they are same..